model-optimizer/extensions/front/onnx/ONNXResize10ToInterpolateV4.py

```diff
@@ -0,0 +1,131 @@
+    log.debug("Converting of ONNX Resize-10 to Interpolate-4 "
+              "is triggered for node {}.".format(resize.soft_get('name', resize.id)))
+
+    resize_name = resize.soft_get('name', resize.id)
+
+    rank_node = Rank(graph, {'name': resize_name + '/rank_'}).create_node()
+    range_node = create_op_with_const_inputs(graph, Range, {0: int64_array(2), 2: int64_array(1)},
+                                             {'name': resize_name + '/rank_'})
+
+    sizes_ss = create_op_with_const_inputs(graph, StridedSlice,
+                                           {1: int64_array([2]),
+                                            2: int64_array([0]),
+                                            3: int64_array([1])},
+                                           {'name': resize_name + '/sizes_ss_',
+                                            'begin_mask': int64_array([1]),
+                                            'end_mask': int64_array([0]),
+                                            'new_axis_mask': int64_array([0]),
+                                            'shrink_axis_mask': int64_array([0]),
+                                            'ellipsis_mask': int64_array([0])})
+    scales_ss = create_op_with_const_inputs(graph, StridedSlice,
+                                            {1: int64_array([2]),
+                                             2: int64_array([0]),
+                                             3: int64_array([1])},
+                                            {'name': resize_name + '/scales_ss_',
+                                             'begin_mask': int64_array([1]),
+                                             'end_mask': int64_array([0]),
+                                             'new_axis_mask': int64_array([0]),
+                                             'shrink_axis_mask': int64_array([0]),
+                                             'ellipsis_mask': int64_array([0])})
+
+    rank_node.out_port(0).connect(range_node.in_port(1))
+
+    interpolate_node = Interpolate(graph, {'version': 'opset4',
+                                           'mode': 'linear_onnx' if resize.mode == 'linear' else 'nearest',
+                                           'coordinate_transformation_mode': 'asymmetric',
+                                           'cube_coeff': -0.75,
+                                           'nearest_mode': 'simple',
+                                           'pads_begin': int64_array([0]),
+                                           'pads_end': int64_array([0]),
+                                           'antialias': 0,
+                                           'shape_calculation_mode': 'scales',
+                                           'in_ports_count': 4}).create_node()
+
+    range_node.out_port(0).connect(interpolate_node.in_port(3))
+    shape_of = Shape(graph, {'name': resize_name + '/ShapeOf_'}).create_node()
+
+    add_node = create_op_with_const_inputs(graph, Add,
```

Comment:
Please, add a comment in the code describing why we add `1e-5` here.

Reply:
Done.